test/distributed/_tensor/experimental/test_tp_transform.py

```diff
@@ -37,11 +44,57 @@ class TensorParallelTest(DTensorTestBase):
     def setUp(self) -> None:
         super().setUp()
 
+    def assert_has_c10d_ops(
+        self, gm: torch.fx.GraphModule, expected_ops_count: Dict[str, int]
+    ) -> None:
+        actual_ops_count: Dict[str, int] = defaultdict(int)
+        for node in gm.graph.nodes:
+            if node.op == "call_function":
+                if "c10d_functional" in str(node.target):
+                    actual_ops_count[str(node.target)] += 1
+        self.assertDictEqual(expected_ops_count, actual_ops_count)
+
     @with_comms
     def test_tensor_parallel_transformation_e2e(self):
         torch.manual_seed(0)
         model = MLPListModule(2).to(device=self.device_type)
         inputs = (torch.randn((10, 12)).to(device=self.device_type),)
+        parallel_strategies: Dict[str, ParallelStyle] = {
+            "mlps.0.0": ColwiseParallel,
```

Comment:
usually in eager mode we do `ColwiseParallel()`, but I guess for graph transform either way is fine as it's just a tag